upgrade_tests/drop_compact_storage_upgrade_test.py

```diff
@@ -0,0 +1,63 @@
+from time import sleep
+
+import pytest
+import logging
+
+from cassandra.protocol import InvalidRequest
+
+from dtest import Tester
+
+since = pytest.mark.since
+logger = logging.getLogger(__name__)
+
+VERSION_30 = 'github:apache/cassandra-3.0'
+VERSION_311 = 'github:apache/cassandra-3.11'
+
+
+@pytest.mark.upgrade_test
+@since('3.0', max_version='3.11')
+class TestDropCompactStorage(Tester):
+    def test_drop_compact_storage(self):
+        """
+        Test to verify that dropping compact storage is not possible prior running `nodetool upgradesstables`. The current
+        solution and test take care only about the local sstables. Global check of the sstables will be added as part of
+        CASSANDRA-15897
```

Comment:
I guess that CASSANDRA-15897 is very close so this comment should be enough. We could also keep the nice test for remote nodes with outdated sstables and mark it with:
```
@pytest.mark.skip(reason='awaiting CASSANDRA-15897')
```
That IMO gives it more visibility but, as I said, CASSANDRA-15897 is close so the current way should be ok.